src/main/kotlin/me/modmuss50/mpp/platforms/curseforge/Curseforge.kt

```diff
@@ -53,6 +53,11 @@ interface CurseforgeOptions : PlatformOptions, PlatformOptionsInternal<Curseforg
     @get:Input
     val apiEndpoint: Property<String>
 
+    // Set the type of changelog for CurseForge. If unset, defaults to MARKDOWN
+    @get:Input
+    @get:Optional
+    val changelogType: Property<CurseforgeApi.ChangelogType>
```

Comment:
Im not sure if using this enum here is the best solution, does it mean you need to import CurseforgeApi into your buildscript? A test to show this off would go a long way.

Reply:
Ah, I prefer importing and using enums instead of strings in general, it means I am less likely to have a typo.

I think this case is similar to `JavaVersion` or `ReleaseType`... but actually I'm not really sure what is going on with `ReleaseType`, the way it's handled seems really odd to me. 

Can you explain to me what you'd like this to be?
For now I can make this a string and add some validation.